src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java

```diff
@@ -76,47 +97,52 @@ public void requestAndWriteKitRequests(List<LatestKitRequest> latestKitRequests)
                                                             String collaboratorParticipantId = KitRequestShipping.getCollaboratorParticipantId(latestKit.getBaseURL(), latestKit.getInstanceID(), latestKit.isMigrated(),
                                                                     latestKit.getCollaboratorIdPrefix(), (String) profile.get("guid"), (String) profile.get("hruid"), kitRequestSettings.getCollaboratorParticipantLengthOverwrite());
 
-                                                            KitRequestShipping.addKitRequests(latestKit.getInstanceID(), kitDetail, kitType.getKitTypeId(),
-                                                                    kitRequestSettings, collaboratorParticipantId);
+                                                            if (kitHasSubKits) {
+                                                                List<KitSubKits> subKits = kitRequestSettings.getSubKits();
+                                                                addSubKits(subKits, kitDetail, collaboratorParticipantId, kitRequestSettings, latestKit.getInstanceID());
+                                                                DDPParticipant ddpParticipant = ElasticSearchUtil.getParticipantAsDDPParticipant(participantsESData, kitDetail.getParticipantId());
+                                                                if (ddpParticipant != null) {
+                                                                    if (StringUtils.isNotBlank(kitRequestSettings.getExternalShipper())) {
+                                                                        orderKit.add(new KitRequest(kitDetail.getParticipantId(), (String) profile.get("hruid"), ddpParticipant));
+                                                                        logger.info("Added kit with external order number " + orderKit.get(orderKit.size() - 1).getExternalOrderNumber() + " to the order listftodo pea");
```

Comment:
listftodo pea ?

Reply:
Yeah i noticed that today, will push the correct one later